cns/networkcontainers/networkcontainers_windows.go

```diff
@@ -188,7 +188,7 @@ func configureNetworkContainerNetworking(operation, podName, podNamespace, docke
 			{k8sPodNameStr, podName}}}
 	log.Printf("[Azure CNS] run time conf info %+v", cniRtConf)
 
-	netConfig, err := getNetworkConfig(netPluginConfig.networkConfigPath)
+	netConfig, err := getNetworkConfig(netPluginConfig.networkConfigPath, netPluginConfig.cnsUrl, true)
```

Comment:
can we check orchestrator type and set to true here..by default in multitenancy case its false

Reply:
I will check in restserver.go